src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-details/iscsi-target-details.component.html

```diff
@@ -8,7 +8,7 @@
       <span>&nbsp;</span>
 
       <span class="label"
-            [ngClass]="{'label-success': ['in', 'up'].includes(node.status), 'label-danger': ['down', 'out'].includes(node.status)}">
+            [ngClass]="{'label-success': ['in', 'up', 'logged_in'].includes(node.status), 'label-danger': ['down', 'out', 'logged_out'].includes(node.status)}">
```

Comment:
Since `node.status` can only be `logged_in` or `logged_out`, I think we can remove the other status.

Reply:
@tspmelo Fixed